content/2019-munich/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Overview
+---
+
+## Overview
+
+PromCon EU 2019 is the fourth conference dedicated to the
```

Comment:
Not EU -> Europe to emphasize the geography rather than the political union?

Reply:
I am torn; EU is the name of the region and I called North America NA. TTBOMK those are the official shorthands.